plugins/repository-s3/build.gradle

```diff
@@ -354,8 +194,8 @@ if (useFixture && minioDistribution) {
     restIntegTestTask.clusterConfig.jvmArgs = jvmArgs
   }
 
-  integTestMinioRunner.dependsOn(startMinio)
-  integTestMinioRunner.finalizedBy(stopMinio)
+  integTestMinioRunner.dependsOn(postProcessFixture)
+  integTestMinioRunner.finalizedBy(composeDown)
```

Comment:
I wouldn't want to use `composeDown` here. We don't yet clean this up other than running `./gradlew clean` which depends on `composeDown`. We plan to make cleanup part of the plugin eventually. 
If we must have this now, we should add an alias to `composeDown` to the plugin so we don't reference it directly, but I would just remove this line.

Reply:
@atorok it seems the container isn't shut down though without this line?

Reply:
Ah ok, well killing that line then :)